src/main/java/de/retest/web/selenium/RecheckDriver.java

```diff
@@ -15,9 +13,8 @@ public RecheckDriver( final RemoteWebDriver wrapped ) {
 		super( wrapped );
 	}
 
-	public RecheckDriver( final RemoteWebDriver wrapped, final RecheckOptions options,
-			final AutocheckingCheckNamingStrategy namingStrategy ) {
-		super( wrapped, options, namingStrategy );
+	public RecheckDriver( final RemoteWebDriver wrapped, final RecheckWebOptions options ) {
```

Comment:
This is also a breaking change.

Reply:
Granted ... but one I'd accept as being in the grey-area...